translators/crate.py

```diff
@@ -2,28 +2,56 @@
 from datetime import datetime, timedelta
 from translators.base_translator import BaseTranslator
 from utils.common import iter_entity_attrs
+import logging
 import statistics
 
+
 # NGSI TYPES: Not properly documented so this might change. Based on experimenting with Orion.
+NGSI_ID = 'id'
+NGSI_TYPE = 'type'
+NGSI_TEXT = 'Text'
+NGSI_DATETIME = 'DateTime'
+
 # CRATE TYPES: https://crate.io/docs/reference/sql/data_types.html
 NGSI_TO_CRATE = {
     "Array": 'array(string)',  # TODO: Support numeric arrays
```

Comment:
regarding array, i think there is no way to do a fixed mapping to a type,
you should check what is the type in the array, and dynamically generate it.
it could be even an object in term of json. a simple alternative could be to use object.

Reply:
right, that's more or less what the TODO is about.

Reply:
Created #36

Reply:
if the object storage works, i would the simplest (and probably more performing) solution.